frontend/src/audio.html

```diff
@@ -145,7 +161,7 @@
               data-tippy-placement="bottom"
             >
               <!-- Share -->
-              <svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke-width="2" stroke="currentColor" class="h-full w-4 pb-1">
+              <svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke-width="2" stroke="currentColor" class="h-full w-3.5">
```

Comment:
You make it smaller than the other two icons. Isn't removing `pb-1` enough? I think it caused the shift. Padding in the bottom.

Reply:
Also, there is a full padding on the parent `span` (`p-1`) which is not there on other elements of this line. Maybe it causes the shift? Please review this.